src/viztracer/modules/vcompressor/vcompressor.c

```diff
@@ -220,6 +229,14 @@ vcompressor_decompress(VcompressorObject* self, PyObject* args) {
 
     parsed_events = load_events_from_file(fptr);
 
+    file_info = load_file_info(fptr);
```

Comment:
I know `load_events_from_file` might be a bit confusing and ambiguous. Here `events` does not mean `traceEvents`. It simply means "the data from the actual json report"(compared to say metadata which is not part of the json report). So `file_info` should also be read in `load_events_from_file`, as `file_info` should be considered as an "event" in the compressed binary file.